src/defaults.js

```diff
@@ -17,6 +17,7 @@ var primary = {
   bookingGraph: 'instant',
   debug: false,
   availabilityView: 'agendaWeek',
+  showTestRibbon: false,
```

Comment:
Again, maybe rename to `showTestModeRibbon` to make it more explicit and avoid confusion. But is there really a need to make this a configurable setting?

Reply:
well, I wanted to give the user more power. but ill remove it for now.